### ISSUE TITLE
Programmatically updating nodes breaks the scripture field

### ISSUE THREAD

**Issue by rudolfbyker** [refstudycentre/scripture#5]
Example code which breaks the scripture field, but works with all other fields:

``` php
  // load the node
  $node = node_load($form_state['build_info']['args'][0]->nid);
  /** @var EntityDrupalWrapper $wrapper An EntityMetadataWrapper for the node */
  $wrapper = entity_metadata_wrapper('node', $node);

  // update some field on the node
  $wrapper->lw_detail->set(20);

  // save the node
  $wrapper->save();
```

**Comment by rudolfbyker** [comment]
When editing the node, and a form was not displayed for it, `$items` looks like this …
![hook_field_presave](https://cloud.githubusercontent.com/assets/10025342/12198727/4b60cfee-b61b-11e5-8413-883c7468bf15.png)
… instead of this …
![hook_field_presave 2](https://cloud.githubusercontent.com/assets/10025342/12198804/d8baa8e2-b61b-11e5-9cec-435acac5657a.png)
… so the `scripture_field_presave` function needs to be smarter.

**Comment by rudolfbyker** [comment]
Fixed in commit bb9581142fe3fd257a37daca8623b7ec031d21f7
